<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(4, 4, 4);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QComboBox" name="puertos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="graficawidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1261</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="guardarButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>GUARDAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre_persona">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 254, 254);</string>
    </property>
    <property name="text">
     <string> Nombre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cargarButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CARGAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="espectrogramaButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>WAVELET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections/>
</ui>
